nautobot/docs/configuration/optional-settings.md

```diff
@@ -910,6 +899,22 @@ in order for the repository to sync.
 
 ---
 
+## NAUTOBOT_LOG_DEPRECATION_WARNINGS
+
++++ 1.5.2
+
++/- 1.5.3
+    This was previously available as a config file setting but changed to environment-variable only. Also `DEBUG = True` will no longer work to log deprecation warnings.
```

Comment:
Just remembered that there's a "Deprecation Warnings Silenced by Default" entry in the `version-1.5.md` release-notes overview that also needs to be updated to reflect this fix.

Reply:
Updated.